dmoj/urls.py

```diff
@@ -115,6 +115,8 @@ def paged_list_view(view, name):
 
     url(r'^problems/$', problem.ProblemList.as_view(), name='problem_list'),
     url(r'^problems/random/$', problem.RandomProblem.as_view(), name='problem_random'),
+    url(r'^problems/suggest_list/$', problem.SuggestList.as_view(), name='problem_suggest_list'),
+    url(r'^problem/suggest$', problem.ProblemSuggest.as_view(), name='problem_suggest'),
```

Comment:
if we use `^problem/suggest$`  url, then the site cannot have the problem with code `suggest`

Reply:
Anyway, is there any button that can lead me to this URL other than type it?

Reply:
I'm téting that with the navbar submenu. Maybe a button at the problemset?

Reply:
a button at the problem set seems good